src/main/java/me/botsko/prism/Prism.java

```diff
@@ -895,9 +895,10 @@ public void actionRecorderTask() {
 	 */
     public void launchScheduledPurgeManager() {
         final List<String> purgeRules = getConfig().getStringList( "prism.db-records-purge-rules" );
+        Integer purgeCycleTime = Math.round(24/getConfig().getInt("prism.purge.perday",2));
```

Comment:
I thought it best to implicitly declare it ...to ensure we dont get casting errors later on... but we can go back and rely on type casting